docs/pages/2020_Intro_Week/index.rst

```diff
@@ -23,11 +23,75 @@ This is the content of the `AiiDA virtual tutorial <http://www.aiida.net/aiida-v
 Presentations
 -------------
 
-.. Add links to presentations
+For this virtual tutorial, all presentations have been recorded beforehand, so you can watch them at the time that is most convenient for you.
+However, you are required to watch certain presentations before the hands-on sessions of certain days.
+These are tabulated below:
+
++--------------------+--------------------------------------------------------------------+
+| Tuesday July 7th   | `AiiDA Tutorial Overview`_                                         |
++                    +--------------------------------------------------------------------+
+|                    | `Introduction to AiiDA`_                                           |
++                    +--------------------------------------------------------------------+
+|                    | `Setting up an environment with Quantum Mobile Cloud`_             |
++--------------------+--------------------------------------------------------------------+
+| Wednesday July 8th | `Provenance and workflows in AiiDA`_                               |
++--------------------+--------------------------------------------------------------------+
+| Thursday July 9th  | `The AiiDA Plugin Ecosystem`_                                      |
++--------------------+--------------------------------------------------------------------+
+| Friday July 10th   | `The Materials Cloud`_                                             |
++                    +--------------------------------------------------------------------+
+|                    | `AiiDA lab`_                                                       |
++                    +--------------------------------------------------------------------+
+|                    | `AiiDA and OPTIMADE`_                                              |
++                    +--------------------------------------------------------------------+
+|                    | `OSSCAR`_                                                          |
```

Comment:
I think you should add the presenter's names here as well, although it not a hard requirement from by side.

E.g., something like:
```rst
`AiiDA Tutorial Overview`_ by Marnik Bercx
```

Reply:
I added another column for this before the title, just as for the plugin developer talks.

One issue is that the table doesn't fit in the width of the page now, so they need to scroll to see the full title of @chrisjsewell's talk. :)

Reply:
None of the two tables fit in the width for me now.
Would it not be possible to wrap the text and fix the width?

Reply:
Possibly, but I think you'd have to add some CSS for this, which I'd have to look up. ^^ If you know how to do this, I'd be happy to change it.

Reply:
A quick search online tells me I don't know how to do this quickly.

Reply:
However, it seems you can make the table less wide by simply removing unused white-space in a column? Always needing a minimum of 1 white-space at each end of course.

Reply:
Hmm, I'm not sure that would do the trick. In the rendered table, there is barely any white space left atm. 

How about I add this issue to a checklist and we have a look at it later? :) For now I'd like to merge this into `2020-intro-week` so we can merge into `master` asap.